packages/browser-wallet/src/popup/pages/Account/AccountActions/AccountActions.tsx

```diff
@@ -49,11 +51,13 @@ function ActionLinks({ children }: ActionLinksProps) {
     );
 }
 
-export default function AccountActions() {
+type Props = ClassName;
+
+export default function AccountActions({ className }: Props) {
     const { t } = useTranslation('account', { keyPrefix: 'actions' });
 
     return (
-        <nav className="account-page-actions">
+        <nav className={clsx('account-page-actions', className)}>
```

Comment:
I don't really understand why you need this className parameter (which you use to specify the flex), instead of just using `account-page-actions` for that. (Also is it not only going to used in this context?

Reply:
It's to specify `height` in the most recent commit, and it's because the account page uses the heights of the different elements on page to ultimately calculate the height of the scrollable area on the page (f.x. transaction log). Having all the different heights in the same css file makes it easier to understand what's going on.